consensus/src/state_computer.rs

```diff
@@ -60,11 +67,25 @@ impl ExecutionProxy {
                 callback();
             }
         });
+        let (commit_tx, mut commit_rx) =
+            channel::new::<CommitType>(10, &counters::PENDING_QUORUM_STORE_COMMIT_NOTIFICATION);
+        let notifier = commit_notifier.clone();
+        handle.spawn(async move {
+            while let Some((epoch, round)) = commit_rx.next().await {
+                if let Err(e) =
+                    monitor!("notify_commit", notifier.notify_commit(epoch, round).await)
```

Comment:
not for this pr but we probably should unify the commit notification with mempool. currently it's routed through state sync.

Reply:
Why do we spawn a thread for this?